curation/fields.py

```diff
@@ -574,6 +576,7 @@ def __init__(self, *args, **kwargs):
             kwargs['choices'] = ContentTypeIdChoices(self.ct_choices)
         kwargs.pop('to', None)
         self.source_field_name = kwargs.pop('source_field', None)
+        kwargs.setdefault('on_delete', models.CASCADE)
```

Comment:
Will this cascade delete will remove entries from the django.contenttypes table?

Reply:
talked in real life with David - this sets a default / matches previous default behaviors before `on_delete` became required.